src/contentful/utils.js

```diff
@@ -36,10 +42,22 @@ export const getContentfulRequest = (type, contentQuery) => {
   };
 };
 
+const hasValidationErrors = (payload) => payload?.errors;
```

Comment:
will we ever need to overwrite that method (in case of more validation logic)? if so, maybe add as parameter to function below?

Reply:
Don't think apps should ever do further validation. Content validation should happen in contentful to give users proper quick feedback, therefore I don't expect to need this function any time soon.